DuckDuckGo/BookmarksViewController.swift

```diff
@@ -444,6 +438,8 @@ class BookmarksViewController: UIViewController, UITableViewDelegate {
                 completion(true)
             }
             present(alertController, animated: true)
+        } else if bookmark.isFolder {
+            delete()
```

Comment:
This is to avoid the debug assertion, right?
Why aren't we calling the completion handler here?

Reply:
Good catch, I'll check that.

Reply:
There's a few paths that don't but I think this one should so I've added it.  I had a quick look and can't really see what the implication of not calling it is.  It seems that if you don't then the buttons shown on swipe don't get hidden, but since we're removing the row anyway it doesn't matter *too* much.